modules/tinymce/changelog.txt

```diff
@@ -12,6 +12,7 @@ Version 5.7.0 (TBD)
     Changed the type signature for `editor.selection.getRng()` incorrectly returning `null` #TINY-6843
     Changed some `SaxParser` regular expressions to improve performance #TINY-6823
     Fixed table width style was removed when copying #TINY-6664
+    Fixed content containing corrupted base64 URLs in style attributes after parsing #TINY-6828
```

Comment:
Maybe `Fixed base64 URLs used in style attributes were corrupted when parsing HTML #TINY-6828`?

Reply:
Thanks! I couldn't get this to sound right over the weekend.
```suggestion
    Fixed base64 URLs used in style attributes were corrupted when parsing HTML #TINY-6828
```